src/main.c

```diff
@@ -288,6 +291,46 @@ static uint32_t read_encoder_right(void)
 
 
 /**
+ * @brief Setup for adc: Four injected channels on scan mode for ADC1.
+ * - Initialize channel_sequence structure to map physical channels vs
+ * sw injected channels.
+ * - Power off the ADC to be sure that doesn't run during configuration.
+ * - Enable scan mode with single conversion mode triggered by software.
+ * - Configure the alignment (right) and the sample time (28.5 cycles of ADC
+ * clock)
+ * - Set injected sequence with channel_sequence structure
+ * - Power on the ADC and wait for ADC starting up (about 3 ms, See adc_power_on
```

Comment:
Maybe this "See ..." could be removed from here and added at the end in a "@see ..."?

Reply:
See deleted